cluster.c

```diff
@@ -1132,6 +1149,7 @@ RRStatus computeHashSlot(RedisRaftCtx *rr, RaftReq *req)
             } else {
                 if (slot != thisslot) {
                     RedisModule_Free(keyindex);
+                    RedisModule_ReplyWithError(req->ctx, "CROSSSLOT Keys in request don't hash to the same slot");
```

Comment:
moved ReplyWithError here, so we can return others errors as well in computeHashSlot in future.

Reply:
So maybe rename it to computeHashSlotOrReplyError?

Reply:
ok